lib/wahwah/mp4/atom.rb

```diff
@@ -44,7 +44,6 @@ def self.find(file_io, *atom_path)
       # This can often be usefully treated as a four-character field with a mnemonic value .
       def initialize
         @size, @type = @file_io.read(HEADER_SIZE)&.unpack("Na4")
-        return unless valid?
```

Comment:
I think this was returning too early -- if the size is 1, then `valid?` will be false, and it will return back nil, which I don't think we want.